odoo/addons/base/models/res_company.py

```diff
@@ -131,6 +134,12 @@ def _compute_logo_web(self):
             img = company.partner_id.image_1920
             company.logo_web = img and base64.b64encode(tools.image_process(base64.b64decode(img), size=(180, 0)))
 
+    @api.depends('logo')
```

Comment:
Logo is not stored, which means you will end up computing 'uses_default_logo' each time 'logo' is computed ... you should not have a stored field with unstored triggers. Can't we use logo_web as a trigger ?

Reply:
@tde-banana-odoo  
Hadn't considered related is actually just a non-stored compute...
I'll just depend on `partner_id.image_1920` then, seems to make more sense for this.

It's just that if we change the way 'logo' is computed, the new field might not be accurate.
I've added a test just to make sure it does re-compute, even though it's unlikely to change.

(Added a small motivation at the end of the second commit message too)